test/testdata/workspaces/javascript-sample-app/docker/Dockerfile

```diff
@@ -1,6 +1,9 @@
 FROM node:16-alpine
 
-COPY dist /node
+COPY src /node/
+COPY node_modules /node/
+COPY package.json /node/
+COPY package-lock.json /node/
```

Comment:
I believe this could be simplified to `COPY src node_modules package.json package-lock.json /node/`